ember-cli-build.js

```diff
@@ -4,7 +4,19 @@ const GlimmerApp = require('@glimmer/application-pipeline').GlimmerApp;
 
 module.exports = function(defaults) {
   let app = new GlimmerApp(defaults, {
-    // Add options here
+    minifyJS: {
```

Comment:
This isn't needed (but its fine), we just need to update to `@glimmer/application-pipeline@0.7.2` (which includes these defaults as of https://github.com/glimmerjs/glimmer-application-pipeline/pull/106).